sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/models/TrainingOptions.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.training.models;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.Duration;
+
+/**
+ * Options that may be passed when using training APIs on Form Training client.
+ */
+@Fluent
+public class TrainingOptions {
```

Comment:
Make this final.

Reply:
this class is extendable.  is that correct @samvaity